alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
+
+COPY etc /etc
+
+RUN rm -r /etc/sv/rails /etc/service/rails \
+  && /root/setup-directories.sh root r /etc/service/postgres /etc/service/buildkite \
+  && /root/setup-directories.sh postgres rw /var/db/postgresql \
+  && ln -s -- /etc/sv/postgres /etc/service/postgres/run \
+  && ln -s -- /etc/sv/buildkite /etc/service/buildkite/run \
+  && echo '@testing-extra https://s3.amazonaws.com/alpine-packages/testing-extra' >> /etc/apk/repositories \
```

Comment:
AFAIK: I'd need to recompile the packages to change it back to `@testing`. Will do that tomorrow. The official [documentation](http://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#Creating_an_APKBUILD_file) said "new packages should be added to @testing". 

> If you doubt to which repository your package belongs to you can safely use testing.